packages/mobile/src/exchange/ExchangeHomeScreen.tsx

```diff
@@ -160,53 +160,55 @@ function ExchangeHomeScreen({ navigation }: Props) {
         stickyHeaderIndices={[]}
         contentContainerStyle={styles.contentContainer}
       >
-        <DisconnectBanner />
-        <View style={styles.goldPrice}>
-          <View style={styles.goldPriceTitleArea}>
-            <Text style={styles.goldPriceTitle}>{t('goldPrice')}</Text>
-            <Touchable onPress={navigateToGuide} hitSlop={variables.iconHitslop}>
-              <InfoIcon size={14} />
-            </Touchable>
-          </View>
-          <View style={styles.goldPriceValues}>
-            <Text style={styles.goldPriceCurrentValue}>
-              {currentGoldRateInLocalCurrency
-                ? displayLocalCurrency(currentGoldRateInLocalCurrency)
-                : '-'}
-            </Text>
-
-            {rateChangeInPercentage && (
-              <Text style={rateWentUp ? styles.goldPriceWentUp : styles.goldPriceWentDown}>
-                {rateWentUp ? '▴' : '▾'} {rateChangeInPercentage.toFormat(2)}%
+        <SafeAreaView style={styles.background} edges={['bottom']}>
```

Comment:
Why do we have a safeAreaView inside a safeAreaView?

Reply:
Well, this is my workaround, when we have to use `SafeAreaView` inside `ScrollView`. Cause we need one part (top) of `SafeAreaView` be outside of `ScrollView` and another (bottom) inside `ScrollView`. It does look odd, I agree, do you know how we can solve it better?